.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,24 @@
 ---
-name: 'Pull Request: New issue'
-about: Report and propose a fix for a (small) issue in Hazel
```

Comment:
Uppercase the letter 'a', because it is the first letter in the sentence

Reply:
I believe it's already uppercase.

Reply:
If you are talking about the `about:` tag I think that needs to be lowercase for GitHub to detect it properly.

Reply:
the name and about are part of the YAML interpretation of github, this is not part of the actual PR template, this is github stuff...